src/screens/LoginPage/LoginPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { getAuth, signInWithEmailAndPassword } from 'firebase/auth';
+import { FC } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '../../components/Button';
+import CustomInput from '../../components/CustomInput';
+import { useAppDispatch } from '../../hooks/useAppDispatch';
+import { useForm } from '../../hooks/useForm';
+import AuthLayout from '../../Layout/Auth';
+import { IUser } from '../../models/IUser';
+import { setUser } from '../../store/slices/userSlice';
+import { Wrapper } from './StyledLoginPage';
+
+
+const LoginPage: FC = () => {
+	const { login, password, changeLogin, changePassword } = useForm();
+	const dispatch = useAppDispatch();
+	const auth = getAuth();
+	const navigate = useNavigate();
+
+
+	const handleLogin = (email: string, password: string) => {
+		signInWithEmailAndPassword(auth, email, password)
+			.then(({ user }) => {
+				// @ts-ignore
```

Comment:
why ignore?

Reply:
becouse TS doesn`t see accessToken in user response